components/_global-navigation.scss

```diff
@@ -200,17 +200,26 @@ $notification-counter-size: 18px;
 	}
 
 	.wds-search-is-active {
-		.wds-global-navigation__search-input-wrapper {
-			margin-right: 15px;
-		}
+		.wds-global-navigation__search {
+			flex: 2;
 
-		.wds-global-navigation__search-input::placeholder {
-			color: $color-light-gray;
-		}
+			&-input-wrapper {
```

Comment:
It's really hard to be grepped now. What do you think about using this pattern only after double-underscore?